python/Ganga/Core/InternalServices/Coordinator.py

```diff
@@ -52,9 +52,10 @@ def isCredentialRequired(credObj):
         return Workspace_runtime.requiresAfsToken() or Repository_runtime.requiresAfsToken()
 
     if getName(credObj) == 'GridProxy':
-        from Ganga.GPI import jobs, typename
+        from Ganga.Core.GangaRepository import getRegistryProxy
+        from Ganga.Runtime.GPIFunctions import typename
         from Ganga.GPIDev.Base.Proxy import stripProxy
-        for j in jobs:
+        for j in getRegistryProxy('jobs'):
```

Comment:
Does this need to be a proxy-wrapped object. Could we not just access the job registry object directly?